var/spack/repos/builtin/packages/boxmg4wrf/package.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Boxmg4wrf(MakefilePackage):
+    """This is a version of the Dendy-Moulton "Black Box Multigrid" elliptic
+    equation solver (until a more permanent archive is established). It has
+    been updated to allow single-precision solutions and is posted here for
+    use with an upcoming release of WRF (Weather Research and Forecasting)
+    model including storm electrification. Only runs under MPI at this time
+    (i.e., no SMP/OpenMP support)."""
+
+    homepage = "https://sourceforge.net/projects/boxmg4wrf/"
+    git      = "https://git.code.sf.net/p/boxmg4wrf/git"
+
+    parallel = False
+
+    # maintainers = ['github_user1', 'github_user2']
```

Comment:
This should be removed if you don't want to add yourself as a reviewer by including your GitHub account in the list.

Reply:
Done